.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish to PyPI and TestPyPI
+
+on: push
+
+jobs:
+  build-and-publish:
+    name: Build to PyPI and TestPyPI
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Build distribution package
+      run: |
+        pip install twine wheel
+        pip wheel -w dist --no-deps .
+    - name: Publish to Test PyPI
```

Comment:
What's your reasoning behind running this upload to the test PyPI instance on every commit, and not just on commits to `master`?

Reply:
It seems useful to me to have a way to test that the upload works (by incrementing the version) before merging a branch into master!

Reply:
Ah, I see. Many projects handle that case by using branches prefixed with `release/` for PRs that are incrementing version numbers, and then restricting test CI uploads to `release/*` branches. What do you think about that solution? I only ask because it seems kind of spammy to try to run a PyPI upload on every commit, and it introduces a potential human error where someone messes with the version number in a branch commit and doesn't realize that doing so will actually upload a new distribution to the test PyPI instance.

Reply:
Hm, I see your point. Does that mean that a branch without `release` in the name would never upload to Test PyPI? It feels to me that for this scale of project we may as well stick to uploading a test distribution whenever a change is merged to `master` instead of introducing that complexity to branch naming. If you think that restricting test uploads to `master` is the best pattern here I trust your judgement.

Reply:
Yeah, it seems to me like restricting testing deployments to merging with `master` is our best option. I think you're right that it's not ideal for testing releases, but at least it follows our typical pattern of how we deploy to staging sites. I'd be interested in having a larger conversation with Hannah and Forest about standardizing this workflow since it seems like a problem we'll have with any public Python package we publish.

Reply:
Ok this action should now only run for `push` events to `master`. Once this looks good to you, @jeancochrane, I'll merge it and confirm it works. One thing I went back and forth on was whether to restrict the whole Action to pushes to `master` or just the `Publish to Test PyPI` step. After some digging around and reading through [this thread](https://github.community/t5/GitHub-Actions/pull-request-action-does-not-run-on-merge/td-p/29284) I picked the first option. I have the sense that it would be better to write a separate Action if we later want anything to run on every commit (eg, tests (#4))

Reply:
Good thinking, I agree 👍